server/database/models/index.js

```diff
@@ -13,10 +13,13 @@ db.stores = require('@database/models/stores').model(sequelize, Sequelize.DataTy
 db.addresses = require('@database/models/addresses').model(sequelize, Sequelize.DataTypes);
 db.suppliers = require('@database/models/suppliers').model(sequelize, Sequelize.DataTypes);
 db.users = require('@database/models/users').model(sequelize, Sequelize.DataTypes);
+db.users = require('@database/models/students').model(sequelize, Sequelize.DataTypes);
+db.users = require('@database/models/subjects').model(sequelize, Sequelize.DataTypes);
```

Comment:
Why is this db.users ? Should be students and subjects right ?

Reply:
Updated accordingly.